experiments/interpretation/dutchf3_patch/local/train.py

```diff
@@ -158,12 +152,16 @@ def run(*options, cfg=None, debug=False):
         augmentations=val_aug,
     )
```

Comment:
Related to the comment above. We might want to keep the augmentation as it is. 
```suggestion
)
basic_aug = Compose(
        [
            Normalize(mean=(config.TRAIN.MEAN,), std=(config.TRAIN.STD,), max_pixel_value=1),
            PadIfNeeded(
                min_height=config.TRAIN.PATCH_SIZE,
                min_width=config.TRAIN.PATCH_SIZE,
                border_mode=0,
                always_apply=True,
                mask_value=255,
                value=0,
            ),
            Resize(
                config.TRAIN.AUGMENTATIONS.RESIZE.HEIGHT, config.TRAIN.AUGMENTATIONS.RESIZE.WIDTH, always_apply=True,
            ),
            PadIfNeeded(
                min_height=config.TRAIN.AUGMENTATIONS.PAD.HEIGHT,
                min_width=config.TRAIN.AUGMENTATIONS.PAD.WIDTH,
                border_mode=0,
                always_apply=True,
                mask_value=255,
            ),
        ]
    )
```

Reply:
discussed - reverting back to older code